source/_templates/registrations/windows/installation_directory.rst

```diff
@@ -0,0 +1,12 @@
+.. Copyright (C) 2020 Wazuh, Inc.
```

Comment:
This information is useful outside of the registration guide. Path: ` source/_templates/windows/installation_directory.rst`

Reply:
Done